README.md

```diff
@@ -146,6 +146,60 @@ optional arguments:
                         DynamoDB table to use for credential storage
 ```
 
+## IAM Policies
+
+### Secret Writer
+In order to put or write secrets to credstash, and you are not using KMS Key Grants or Key Policies, at least the following permissions are required:
+```
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "kms:GenerateDataKey"
+      ],
+      "Effect": "Allow",
+      "Resource": "arn:aws:kms:us-east-1:AWSACCOUNTID:key/KEY-GUID"
+    },
+    {
+      "Action": [
+        "dynamodb:PutItem"
+      ],
+      "Effect": "Allow",
+      "Resource": "arn:aws:dynamodb:us-east-1:AWSACCOUNTID:table/credential-store"
+    }
+  ]
+}
+```
+If you are using Key Policies or Grants, then the `kms:GenerateDataKey` is not required in the policy for the IAM user/group/role. Replace `AWSACCOUNTID` with the account ID for your table, and replace the KEY-GUID with the identifier for your KMS key (which you can find in the KMS console). 
+
+### Secret Reader
+In order to `get` or read a secret from credstash (including `getall`), and you are not using KMS Key Grants or Key Policies, you must have at least these permissions:
```

Comment:
Maybe go with something like

"You can read secrets from credstash with the `get` or `getall` actions by either using KMS Key Grants, KMS Key Policies, or IAM Policies. If you are using IAM Policies, the following IAM permissions are the minimum required to be able to put or read secrets"